src/Kongverge/DTOs/KongRoute.cs

```diff
@@ -87,19 +98,27 @@ public virtual async Task Validate(IReadOnlyCollection<string> availablePlugins,
                 errorMessages.Add("At least one of Route 'Hosts', 'Methods', or 'Paths' must be set.");
             }
 
-            if (Hosts == null || Hosts.Any(x => string.IsNullOrWhiteSpace(x) || x.StartsWith("*.") && x.EndsWith(".*") || Uri.CheckHostName(RemoveWildcards(x)) == UriHostNameType.Unknown))
+            if (Hosts?.Any(string.IsNullOrWhiteSpace) == true)
+            {
+                errorMessages.Add("Route Hosts is invalid (cannot contain null or empty values).");
+            }
+            if (Hosts?.Any(x => !string.IsNullOrWhiteSpace(x) && x.StartsWith("*.") && x.EndsWith(".*")) == true)
+            {
+                errorMessages.Add("Route Hosts is invalid (values cannot begin and end with a wildcard, only one wildcard at the start or end is allowed).");
+            }
+            if (Hosts?.Any(x => !string.IsNullOrWhiteSpace(x) && Uri.CheckHostName(RemoveWildcards(x)) == UriHostNameType.Unknown) == true)
```

Comment:
nit picking, needs a blank line

Reply:
This was intentional, as this batch of if-statements are all checking the same property.